Assets/Scripts/PlayerController/PlayerMovement.cs

```diff
@@ -37,6 +37,7 @@ public class PlayerMovement : MonoBehaviour
     private float m_LandingTime = 0;
     private bool m_IsMidAir;
     private bool m_JumpedInCurrentFrame;
+    private bool m_cannotMove;
```

Comment:
Let's follow the boolean naming convention, starting the word with a `is` or `has` etc, and keeping it in positive form (can instead of cannot, etc). I suggest `m_IsMovementLocked`

Reply:
Okay the player movement scripts were penned by Sean